### PR TITLE
chore(rustfmt): format `Cargo.toml`

### PR DESCRIPTION
**Pull request by xxchan**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

Draft. There seems to be some minor problems to be fixed.

similar to https://github.com/risingwavelabs/risingwave/pull/9409

test https://github.com/rust-lang/rustfmt/pull/5240

## Documentation

- [x] My PR **DOES NOT** contain user-facing changes.

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

**Comment by TennyZhuang** [comment]
Can we check that in CI?

File: scripts/source/prepare_ci_pubsub/Cargo.toml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 name = "prepare_ci_pubsub"
 version = "0.1.0"
 edition = "2021"
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```
Comment (richardchien) [inline scripts/source/prepare_ci_pubsub/Cargo.toml:6]: Seems this line can be removed.

**Review by richardchien — COMMENTED** [review]
What about https://taplo.tamasfe.dev/? It's used by [Even Better TOML](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml), a VS Code extension. I've used this ext for a long time and it formats well.

**Comment by xxchan** [comment]
I've also used taplo before. It doesn't have any problems, but just lacks
some features specified in the Cargo.toml style guide. TBH I don't care
that much about the formatting now :p.  I'm just more interested in testing
the rustfmt PR.

**Pull request closed without merging**